weldx/core.py

```diff
@@ -1006,7 +1009,8 @@ def __call__(self, **kwargs) -> GenericSeries:
                     v, dims=self._symbol_dims[k], coords={self._symbol_dims[k][0]: v.m}
                 )
             else:
-                v = v.m
+                ref_unit = self._data.coords[k].attrs["units"]
```

Comment:
```suggestion
                ref_unit = self._data.coords[k].pint.units
```
can you do a check check if this works @vhirtham ?

Reply:
Well, let's see what happens if I commit this 😉

Reply:
Doesn't seem to work :(
The tutorial notebook crashes in code cell 10.
If i print the outputs as follows: 

~~~ python
ref_unit = self._data.coords[k].pint.units
print(ref_unit)
print(self._data.coords[k].attrs.get("units", ""))
v = xr.DataArray(v.to(ref_unit), dims=[k]).pint.dequantify()
~~~

I get:

~~~
None
s
~~~

So unfortunately, I guess the `units` property doesn't work if the unit is stored in the attributes.

Reply:
yes I realized now too, that's a bummer